### DISCUSSION TITLE
Clarification on Project folder structure.

### DISCUSSION THREAD

**Discussion by alteredtech** [Allar/ue4-style-guide#96] [Q&A]
The outline says to use `/Content/<Project Name>/everything else`. When UE5 creates a project it uses this structure. `D:/Projects/<Project Name>/Content`.

Following your structure we should make another folder with the project name under content so it looks like the following? `Projects/<Project Name>/Content/<Project Name>/everything else`

**Comment by lucastucious** [comment]
Yes, this allow you to separate marketplace's assets from your game's assets. It simplify migrate process either
